services/slam/builtin/cartographer_int_test.go

```diff
@@ -22,27 +25,54 @@ const (
 	cartoSleepMs = 100
 )
 
-// Checks the cartographer position and map.
-func testCartographerPositionAndMap(t *testing.T, svc slam.Service) {
+// Checks the cartographer map.
+func testCartographerMap(t *testing.T, svc slam.Service) {
 	t.Helper()
 
-	position, err := svc.Position(context.Background(), "test", map[string]interface{}{})
-	test.That(t, err, test.ShouldBeNil)
-	// Typical values for 2D lidar are around (-0.004, 0.004, 0) +- (0.001, 0.001, 0)
-	t.Logf("Position point: (%v, %v, %v)",
-		position.Pose().Point().X, position.Pose().Point().Y, position.Pose().Point().Z)
-	// Typical values for 2D lidar are around (0, 0, -1), theta=0.001 +- 0.001
-	t.Logf("Position orientation: RX: %v, RY: %v, RZ: %v, Theta: %v",
-		position.Pose().Orientation().AxisAngles().RX,
-		position.Pose().Orientation().AxisAngles().RY,
-		position.Pose().Orientation().AxisAngles().RZ,
-		position.Pose().Orientation().AxisAngles().Theta)
 	actualMIME, _, pointcloud, err := svc.GetMap(context.Background(), "test", "pointcloud/pcd", nil, false, map[string]interface{}{})
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, actualMIME, test.ShouldResemble, "pointcloud/pcd")
 	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThanOrEqualTo, 100)
 }
 
+// Checks the cartographer position.
+func testCartographerPosition(t *testing.T, svc slam.Service) {
+	t.Helper()
```

Comment:
Why is it desirable to make `testCartographerPosition` a helper function? I ask b/c when I run tests on my local machine, the tests fail within `testCartographerPosition`, the error message just says that it failed on line 130 (where `testCartographerPosition` was called), but it doesn't tell me the specific `test.That` call that failed. Where as removing the `t.Helper()` does provide me that information.

Reply:
spoke offline but agree shouldn't be a `t.Helper()`